.github/workflows.src/tests-pg-versions.tpl.yml

```diff
@@ -63,21 +63,30 @@ jobs:
       if: ${{ matrix.single-mode }}
       shell: python
       run: |
-        import asyncio, asyncpg
+        import asyncio
+        import subprocess
+
+        from edb.server.pgcluster import get_pg_bin_dir
+
         async def main():
```

Comment:
Drop the async, since it's doing a blocking subprocess call and not doing any async stuff

Reply:
It's doing `await get_pg_bin_dir()`, so need the async